src/Jackalope/ObjectManager.php

```diff
@@ -779,9 +781,17 @@ protected function pathArrayToPropertiesIterator($propertyPaths)
      *
      * @param  string  $cnd         a string with cnd information
      * @param  boolean $allowUpdate whether to fail if node already exists or to update it
-     * @return bool    true on success
+     * @return bool|\Iterator    true on success or \Iterator over the registered node types if repository is not able to process
```

Comment:
Ok, think we must repair interface because bool return type is defined here:

![snimek obrazovky 2016-09-23 v 14 46 11](https://cloud.githubusercontent.com/assets/7081340/18786153/b769059c-819c-11e6-8588-14cf89f5738b.png)